src/convolution/convolution_2d.jl

```diff
@@ -0,0 +1,1153 @@
+### A Pluto.jl notebook ###
+# v0.19.25
+
+#> [frontmatter]
+#> title = "Images and Filtering"
+#> tags = ["images", "filtering", "gaussian", "pixel", "convolution"]
+#> description = "Learn about filters in image processing"
+
+using Markdown
+using InteractiveUtils
+
+# This Pluto notebook uses @bind for interactivity. When running this notebook outside of Pluto, the following 'mock version' of @bind gives bound variables a default value (instead of an error).
+macro bind(def, element)
+    quote
+        local iv = try Base.loaded_modules[Base.PkgId(Base.UUID("6e696c72-6542-2067-7265-42206c756150"), "AbstractPlutoDingetjes")].Bonds.initial_value catch; b -> missing; end
+        local el = $(esc(element))
+        global $(esc(def)) = Core.applicable(Base.get, el) ? Base.get(el) : iv(el)
+        el
+    end
+end
+
+# ╔═╡ 746ff659-88ab-4ff4-8cba-43c798cacd3e
+using Images,TestImages, PlutoUI
+
+# ╔═╡ 04a5c6d4-f8d5-11ed-141a-35481b811ee9
+md"""
+## Images as Lists of numbers
```

Comment:
I would add your name +GH handle on the top here - credit where credit is due :)